packages/elements/src/components/ui/spinner/Spinner.tsx

```diff
@@ -0,0 +1,116 @@
+import { ClassNames, keyframes } from "@emotion/core";
+import * as React from "react";
+import { ReactComponent as SpinnerSvg } from "./spinner-large.svg";
+
+export type SpinnerSize = "large" | "normal" | "small";
+
+export interface SpinnerProps {
+  size?: SpinnerSize | string;
+  color?: string;
+  inverted?: boolean;
+}
+
+const sizes = {
+  large: "78px",
+  normal: "54px",
+  small: "34px",
+  tiny: "20px"
+};
+
+const rotator = keyframes`
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(270deg);
+  }
+`;
```

Comment:
You can use the keywords `to` and `from` as alias for 0% and 100%

Reply:
Oh, ok. That code is copied and pasted from Chris :)